### PR TITLE
Parallel metadiags

### PR DESCRIPTION
**Pull request by mcenerney1**

### CODE REVIEW

**Comment by mcenerney1** [comment]
Complete the display units kludge. All tests pass.

File: src/python/frontend/diags.py [hunk 1]
```diff
@@ -329,9 +329,14 @@ def run_diags( opts ):
                                 fname = os.path.join(outdir,name)
                      
                             if opts['output']['plots'] == True:
-                                makeplots(res, vcanvas, vcanvas2, varid, fname, plot, package)
+                                displayunits = None
+                                if 'displayunits' in opts['output'].keys():
+                                    displayunits = opts['output']['displayunits'][0]
+                                    print '>>>>>>>>>>>>>>>>>>>>displayunits=', displayunits
```
Comment (doutriaux1) [inline src/python/frontend/diags.py]: do we want another print statement? :wink:

**Comment by doutriaux1** [comment]
blessed!

**Comment by mcenerney1** [comment]
No! I'll change it.

**Pull request merged**